pkg/coscheduling/coscheduling.go

```diff
@@ -35,6 +36,17 @@ import (
 	"k8s.io/kubernetes/pkg/scheduler/util"
 )
 
+// Args defines the scheduling parameters for Coscheduling plugin.
+type Args struct {
+	// PermitWaitingTime is the wait timeout in seconds.
+	PermitWaitingTime time.Duration
```

Comment:
As it involves marshal/unmarshal, according to [api-conventions.md#units](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#units),

let's make it `int64`, and rename it as "fooXYZSeconds" ([ref: api-conventions.md#naming-conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#naming-conventions))

(applies to the following parameters as well)

Reply:
Changed as suggested.